### PR TITLE
Scale image preview #290

### PR DESCRIPTION
**Pull request by banasiakadam60**

### CODE REVIEW

File: gnr/ui/appmainwindow.py [hunk 1]
```diff
@@ -20,15 +20,22 @@ def __init__(self):
...
+        """
         if tab_name.lower() == 'new_task':
             self.ui.testTaskButton.setEnabled(enable)
-            self.ui.finishButton.setEnabled(enable)
             self.ui.showAdvanceNewTaskButton.setEnabled(enable)
             self.ui.addResourceButton.setEnabled(enable)
             self.ui.saveButton.setEnabled(enable)
             self.ui.loadButton.setEnabled(enable)
             self.ui.taskTypeComboBox.setEnabled(enable)
+            if success:
```
Comment (badb) [inline gnr/ui/appmainwindow.py]: It's not necessary. If the task is finished properly "start task" button will made enabled by other function from customizer. 
It's also a good practice to keep separate issues in separate commits.

**Comment by badb** [comment]
Seems to behave nicely! But maybe we should remove the frame from the preview?

File: gnr/ui/appmainwindow.py (hunk 1 above)
Comment (banasiakadam60) [inline gnr/ui/appmainwindow.py]: Sorry, I pushed it here by mistake.

**Comment by banasiakadam60** [comment]
OK, removed.

**Comment by banasiakadam60** [comment]
You meant something like that?

File: gnr/ui/appmainwindow.py [hunk 2]
```diff
@@ -15,15 +15,21 @@ def __init__(self):
...
 
         self.ui.setupUi(self.window)
+        self.ui.previewLabel.setFrameStyle(QFrame.NoFrame)
         self.ui.previewLabel.setPixmap(QPixmap(path.join(get_golem_path(), "gnr", "ui", "nopreview.png")))
 
     def show(self):
         self.window.show()
 
-    def setEnabled(self, tab_name, enable):
+    def setEnabled(self, tab_name, enable, success=True):
```
Comment (badb) [inline gnr/ui/appmainwindow.py]: "success" parameter is no longer needed.

**Comment by badb** [comment]
Yes, looks better in my opinion.

**Comment by chfast** [comment]
The code looks good to me too. If everybody happy let's merge that and move on.
